melodic/distribution.yaml

```diff
@@ -1904,6 +1904,12 @@ repositories:
       url: https://github.com/ros-visualization/rqt_runtime_monitor.git
       version: master
     status: maintained
+  rqt_rviz:
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/ros-gbp/rqt_rviz-release.git
+      version: 0.5.9-0
```

Comment:
This is enough to get packages built, but I may suggest adding in the source and doc entries as well?  Similar to how it was done in lunar, here: https://github.com/ros/rosdistro/blob/master/lunar/distribution.yaml#L3790